README.md

```diff
@@ -4,25 +4,41 @@
 
 Gain data model insights quickly during its development and validation phases. This tool is useful when you want to explore your data model for whatever reason; maybe you are creating a complex load script, maybe you want to investigate associations.
 
-![screenshot](./screenshot.png)
+![screenshot](./images/screenshot.png)
 
-## Prerequsities
+# Usage
 
-* Node.js v9+
-* Docker for Mac/Windows
-
-## Get started
-
-Session app with [drugcases.qvf](./data/drugcases.qvf):
+For catwalk to work there are two things needed:
+1. The UI
+2. A Qlik Associative Engine app
 
+## 1. UI
+The UI can either be runned locally using
 ```bash
-ACCEPT_EULA=yes/no docker-compose up -d
 npm install
 npm start
 ```
+Or through the Qlik hosted [catwalk UI](https://catwalk.core.qlik.com)
 
-You may also attach to an existing engine session (make sure that you have already created a session using the same websocket URL + user):
+## 2. Qlik Associative Engine
+You need to provide catwalk with a WS-URL (web socket URL) to the app.  
```

Comment:
Maybe its just enough to type websocket URL and skip the `WS-URL...`?

Reply:
👍